CHANGELOG.md

```diff
@@ -8,17 +8,26 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 # Unreleased
 
+# 4.2.1 (2021-04-28)
+
+## Changed
```

Comment:
There's no suitable kind for dependency bumps, but `changed` felt the most reasonable.